app/lib/katello/resources/candlepin.rb

```diff
@@ -417,7 +417,10 @@ def imports(organization_name)
 
           def pools(key, filter = {})
             if key
-              json_str = self.get(join_path(path(key), 'pools') + hash_to_query(filter), self.default_headers).body
+              params = hash_to_query(filter)
+              params += params == '?' ? '' : '&'
+              params += 'attribute=unmapped_guests_only:!true'
```

Comment:
I would think you could add this as an entry in the filter hash and then just cal hash_to_query,e.g.

```
filter['attribute'] = 'unmapped_guests_only:!true'
params = hash_to_query(filter)
```